charts/lfx-v2-committee-service/templates/ruleset.yaml

```
@@ -0,0 +1,181 @@
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - id: "rule:lfx:lfx-v2-committee-service:committees:create"
+      match:
+        methods:
+          - POST
+        routes:
+          - path: /committees
+      execute:
+        {{- if .Values.authelia.enabled }}
+        - authenticator: authelia
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)